pilot/pkg/serviceregistry/kube/controller/namespacecontroller.go

```diff
@@ -59,6 +60,7 @@ type NamespaceController struct {
 
 // NewNamespaceController returns a pointer to a newly constructed NamespaceController instance.
 func NewNamespaceController(kubeClient kube.Client, caBundleWatcher *keycertbundle.Watcher) *NamespaceController {
+	// This is typically server.istiodCertBundleWatcher
 	c := &NamespaceController{
 		caBundleWatcher: caBundleWatcher,
```

Comment:
Is this comment needed in addition to the above comment?
```suggestion
	c := &NamespaceController{
	        // This is typically server.istiodCertBundleWatcher
		caBundleWatcher: caBundleWatcher,
```